src/YoloDev.Expecto.TestSdk/YoloDev.Expecto.TestSdk.fsproj

```diff
@@ -1,10 +1,9 @@
-﻿<Project Sdk="Microsoft.NET.Sdk; Microsoft.Build.CentralPackageVersions">
+<Project Sdk="Microsoft.NET.Sdk; Microsoft.Build.CentralPackageVersions">
 
   <PropertyGroup>
     <AssemblyName>expecto.visualstudio.dotnetcore.testadapter</AssemblyName>
     <!-- puts build outputs in build folder in nupkg -->
     <BuildOutputTargetFolder>build</BuildOutputTargetFolder>
-    <OutputType>Exe</OutputType>
```

Comment:
Are you sure this is correct? This might need to be conditional. Getting a custom SDK to work is quite finicky.

Reply:
I tested it with net461 and netcoreapp21 in VS 2019, it looks good in the build logs from CI, nunit3testadapter also compiles to the library output type. I feel convinced. I also believe that this, together with not creating net461 assembly, was the source of the issue (test adapters are expected to be `*testadapter.dll` files and `Exe` under net461 compiles to *.exe file ).